common/helm-repositories/mesosphere-repos.yaml

```diff
@@ -40,3 +40,15 @@ spec:
   interval: 10m
   timeout: 1m
   url: "${helmMirrorURL:=https://mesosphere.github.io/dkp-insights-charts-management}"
+---
+apiVersion: source.toolkit.fluxcd.io/v1beta1
+kind: HelmRepository
+metadata:
+  name: mesosphere.github.io-dkp-insights-charts-attached
+  namespace: kommander-flux
+  labels:
+    kommander.d2iq.io/dkp-airgapped: supported
```

Comment:
it looks like this is [added dynamically by one of our controllers](https://github.com/mesosphere/kommander/blob/ae6e771b09cfb2cb83f975260e0415942f87c269/federation/pkg/controllers/helpers/flux.go#L360-L372) for catalog apps, we can drop this (it probably wouldn't break anything but just to avoid confusion)